setup.py

```diff
@@ -36,18 +36,24 @@
 ]
 
 install_requires = [
+    'fastai==0.7.0',
     'Flask~=1.0,>=1.0.2',
     'flask-apispec~=0.0,>=0.7.0',
-    'marshmallow~=3.0.0b13,>=3.0.0b13'
+    'marshmallow~=3.0.0b13,>=3.0.0b13',
+    'spacy~=2.0,>=2.0.0',
+    'torchtext==0.2.3'
 ]
 
 docs_require = []
 
 tests_require = [
+    'fastai==0.7.0',
```

Comment:
you don't have to copy-paste stuff from `install_reqiures` here. These packages will be installed *on top* of the others when installing with `[tests]`.